### DISCUSSION TITLE
Language annotation

### DISCUSSION THREAD

**Discussion by gunnarbittersmann** [schemaorg/schemaorg#3040] [Q&A] [answered]
Hello.

This data
```json
{
	"@type": "TVEpisode",
	"episodeNumber": "S2:E9",
	"name": "wej Duj",
	"inLanguage": "en",
	"partOfSeries": {
		"@type": "TVSeries",
		"name": "Star Trek: Lower Decks"
	},
	"datePublished": "2021-10-07"
}
```
says the language of this episode is English, which is correct.

However, its title is not in English. Is there a way to annotate that the value of `name` (meaning “3 ships”) is in Klingon language (`tlh`)?

**Comment by jaygray0919** [comment]
Link `@TVEpisode` to `@CreativeWork` that sets the title in the target `@Language`.
Remove `inLanguage` property from `@TVEpisode`.
That approach enables expression of `@TVEpisode` in an [array] of multiple languages (one per `CreativeWork` in specific `@Language`)

**Reply by gunnarbittersmann** [reply]
I’m not sure I can follow. Could you provide a code example to demonstrate what you mean, please?

**Comment by chaals** [comment, marked as answer]
Can Schema use the [`localizable`](https://w3c.github.io/string-meta/#use-the-localizable-data-structure) structure developed for JSON after a **lot** of discussion between various groups on how to deal with these use cases?

(I mean, is there any likelihood of implementing it, or do the data consumers not really care that much?)

**Reply by gunnarbittersmann** [reply]
It would look like this then?
```json
{
	"@type": "TVEpisode",
	"episodeNumber": "S2:E9",
	"name": {
		"value": "wej Duj",
		"lang": "tlh"
	},
	"inLanguage": "en"
}
```

BTW, in the WD it’s `language` in some places, `lang` in others. It should be either, but consistent?

**Reply by chaals** [reply]
yes and yes. I am not sure that there is an agreed structure yet, but the point is that it would be helpful to have one (that would be consistent).
